### ISSUE TITLE
Cloud sync questions

### ISSUE THREAD

**Issue by TMiguelT** [ayastreb/money-tracker#39]
I'm interested in how and where the backup of the database is stored in Money Tracker. From looking at the network requests it seems that you're hosting your own CouchDB at https://couch.moneytracker.cc/, and PouchDB replicates onto that. Is that correct? I wonder about the security of that.

Is it possible to configure the URL of the CouchDB replication target to our own CouchDB?

**Comment by ayastreb** [comment]
RIght, currently single CouchDB (https://couch.moneytracker.cc/) is used by
all users of MoneyTracker, which is hosted on a server I pay for myself.

There is currently no UI to change the URL for sync to a different
instance, but it shouldn't be too complicated.

PR implementing this feature would be welcome, but I don't have time myself
right now :)

**Comment by TMiguelT** [comment]
Okay I'll look into doing that, thanks for the reply!

**Comment by TMiguelT** [comment]
I've certainly had a good look, but I'm perplexed by the project structure. I can't find any reference to https://couch.moneytracker.cc/ in the project at all. There is some seemingly relevant config in `src/webtasks/CouchDbProvision.js`, but that reads a `configuration` dictionary that's never defined. Is it put in via pre-processing? It's not in the `.env` file.

**Comment by FedericoNembrini** [comment]
I'd like this feature!
